contract-message-predicate/Cargo.toml

```diff
@@ -10,13 +10,12 @@ rust-version = { workspace = true }
 description = "Script and predicate for relaying contract messages."
 
 [dependencies]
-fuel-asm = "0.31.2"
-fuel-tx = "0.31.2"
+fuel-asm = "0.34.1"
+fuel-tx = "0.31.1"
```

Comment:
These versions used to be in sync but now `tx` has been downgraded by 1 patch. I'm wondering if this is an issue. Should it not stay on `31.2` at least?

Reply:
I needs to match the version been used on the fuels-rs otherwise everything breaks because the types are not compatible

Reply:
https://github.com/FuelLabs/fuels-rs/blob/6efa6f11c8f69f4ef604b70bb78d55f6d4bf76e1/Cargo.toml#L56